packages/shared/lib/core/wallet/utils/getBech32AddressFromAddressTypes.ts

```diff
@@ -2,9 +2,9 @@ import { api } from '@core/profile-manager'
 import { Address, AddressType, AliasAddress, Ed25519Address, NftAddress } from '@iota/sdk/out/types'
 import { getNetworkHrp } from '@core/profile'
 
-export function getBech32AddressFromAddressTypes(address: Address): string | undefined {
+export function getBech32AddressFromAddressTypes(address: Address | undefined): string | undefined {
```

Comment:
Is it necessary to set this `undefined` here?
I think that `?` is enough

Reply:
The fact that he is using `?` means that the address might be undefined at some point, so it's better that the argument type reflects this case